mvrx/build.gradle

```diff
@@ -51,10 +60,19 @@ jacoco {
     toolVersion = "0.8.2"
 }
 
+allprojects {
```

Comment:
I don't think this is needed, allprojects repos are being added in the root build.gradle